Documentation/requirements.txt

```diff
@@ -16,7 +16,7 @@ requests==2.20.0
 six==1.11.0
 snowballstemmer==1.2.1
 Sphinx==1.8.1
-sphinx-rtd-theme==0.2.4
+git+git://github.com/genbit/sphinx_rtd_theme.git#egg=sphinx-rtd-theme
```

Comment:
i'm guessing this is a fork of https://github.com/readthedocs/sphinx_rtd_theme . probably better to put the repo under the cilium github organization.

Reply:
coming soon :)  I just wanted to see it on netlify, that's why PR